message DepthMarketData{


	///交易日
	optional string	TradingDay =1;
	///合约代码
	optional string	InstrumentID=2;
	///交易所代码
	optional string	ExchangeID=3;
	///合约在交易所的代码
	optional string	ExchangeInstID=4;
	///最新价
	optional double	LastPrice=44;
	///上次结算价
	optional double	PreSettlementPrice=5;
	///昨收盘
	optional double	PreClosePrice=6;
	///昨持仓量
	optional int32 PreOpenInterest=7;
	///今开盘
	optional double	OpenPrice=8;
	///最高价
	optional double	HighestPrice=9;
	///最低价
	optional double	LowestPrice=10;
	///数量
	optional int32	Volume=11;
	///成交金额
	optional double	Turnover=12;
	///持仓量
	optional int32	OpenInterest=13;
	///今收盘
	optional double	ClosePrice=14;
	///本次结算价
	optional double	SettlementPrice=15;
	///涨停板价
	optional double	UpperLimitPrice=16;
	///跌停板价
	optional double	LowerLimitPrice=17;
	///昨虚实度
	optional double	PreDelta=18;
	///今虚实度
	optional double	CurrDelta=19;
	///最后修改时间
	optional string	UpdateTime=20;
	///最后修改毫秒
	optional int32	UpdateMillisec=21;
	///申买价一
	optional double	BidPrice1=22;
	///申买量一
	optional int32 BidVolume1=23;
	///申卖价一
	optional double	AskPrice1=24;
	///申卖量一
	optional int32	AskVolume1=25;
	///申买价二
	optional double	BidPrice2=26;
	///申买量二
	optional int32	BidVolume2=27;
	///申卖价二
	optional double	AskPrice2=28;
	///申卖量二
	optional int32	AskVolume2=29;
	///申买价三
	optional double	BidPrice3=30;
	///申买量三
	optional int32	BidVolume3=31;
	///申卖价三
	optional double	AskPrice3=32;
	///申卖量三
	optional int32	AskVolume3=33;
	///申买价四
	optional double	BidPrice4=34;
	///申买量四
	optional int32	BidVolume4=35;
	///申卖价四
	optional double	AskPrice4=36;
	///申卖量四
	optional int32	AskVolume4=37;
	///申买价五
	optional double	BidPrice5=38;
	///申买量五
	optional int32	BidVolume5=39;
	///申卖价五
	optional double	AskPrice5=40;
	///申卖量五
	optional int32	AskVolume5=41;
	///当日均价
	optional double	AveragePrice=42;
	///业务日期
	optional string	ActionDay=43;

}